src/ImageSharp/project.json

```diff
@@ -46,7 +46,31 @@
     "System.Runtime.CompilerServices.Unsafe": "4.0.0"
   },
   "frameworks": {
+    "netstandard1.3": {
```

Comment:
I think it's possible to simplify this. VS2017 projects seem to reference neststandard1.6 for each standard implementation. We'll figure that out when we switch in a couple of weeks though when the RTM hits.